packages/json/src/serialize.ts

```diff
@@ -88,11 +88,11 @@ export function serializeProp(prop: ts.PropertySignature, checker: ts.TypeChecke
     const type = prop.type ? checker.typeToString(checker.getTypeFromTypeNode(prop.type)) : '';
 
     return {
-        name: prop.name.getText(),
+        name: prop.name ? prop.name.getText() : '*',
         kind: 'prop',
         // JSDoc @type will override the true type
         type: jsDocTags.type || type,
-        required: !prop.questionToken,
+        required: prop.name && !prop.questionToken,
```

Comment:
I'm not sure we want to have `required: undefined` in that case, seems good to me to have `required: true`

Reply:
Or maybe we want to have `required: false`? so in that case we want to do:
```
required: prop.name ? !prop.questionToken : false,
```

Reply:
Allright. Done

Reply:
Ahhh, saw your second msg later, ok. just a sec

Reply:
For some reason i tough `undefined && something` will evaluate to `false` but it evaluates to `undefined` . (Oh javascript)
Ok Now is changed as you suggested :+1:  
Wildcards are not necessary required